diff --git a/client/applications/hud/placeable-hud.d.mts b/client/applications/hud/placeable-hud.d.mts
--- a/client/applications/hud/placeable-hud.d.mts
+++ b/client/applications/hud/placeable-hud.d.mts
@@ -59,6 +59,13 @@
      */
     bind(object: PlaceableObject): Promise<void>;
 
+    /**
+     * Toggle the expanded state of the given palette.
+     * @param palette The palette to toggle, or null to collapse the currently expanded palette
+     * @param active  Force the palette to be active or inactive
+     */
+    togglePalette(palette: string | null, active?: boolean): void;
+
     /* -------------------------------------------- */
     /*  Event Listeners and Handlers                */
     /* -------------------------------------------- */
