diff --git a/common/documents/ambient-light.d.mts b/common/documents/ambient-light.d.mts
--- a/common/documents/ambient-light.d.mts
+++ b/common/documents/ambient-light.d.mts
@@ -25,11 +25,15 @@
 export type AmbientLightSchema = {
     /** The _id which uniquely identifies this BaseAmbientLight embedded document */
     _id: fields.DocumentIdField;
+    /** The name used to describe the light */
+    name: fields.StringField;
     /** The x-coordinate position of the origin of the light */
     x: fields.NumberField<number, number, true, false, true>;
     /** The y-coordinate position of the origin of the light */
     y: fields.NumberField<number, number, true, false, true>;
     elevation: fields.NumberField<number, number, true, false, true>;
+    /** The Levels this light is included in. Empty means every Level. */
+    levels: fields.SceneLevelsSetField;
     /** The angle of rotation for the tile between 0 and 360 */
     rotation: fields.AngleField;
     /** Whether or not this light source is constrained by Walls */
