diff --git a/client/config.d.mts b/client/config.d.mts
--- a/client/config.d.mts
+++ b/client/config.d.mts
@@ -35,6 +35,7 @@
     AbstractWeatherShader,
     AdaptiveBackgroundShader,
     AdaptiveColorationShader,
+    AdaptiveDarknessShader,
     AdaptiveIlluminationShader,
     GridShader,
     WeatherShaderEffect,
@@ -94,6 +95,15 @@
 }
 
 /**
+ * A darkness source animation configuration object.
+ */
+export interface DarknessSourceAnimationConfig {
+    label: string;
+    animation: Function;
+    darknessShader?: typeof AdaptiveDarknessShader;
+}
+
+/**
  * Available Weather Effects implementations
  */
 interface WeatherAmbienceConfiguration {
@@ -273,6 +283,7 @@
     Folder: {
         documentClass: typeof documents.Folder;
         collection: typeof collections.Folders;
+        sidebarIcon: string;
     };
 
     /** Configuration for the ChatMessage document */
@@ -620,6 +631,7 @@
 
         objectBorderThickness: number;
         lightAnimations: Record<string, LightSourceAnimationConfig>;
+        darknessAnimations: Record<string, DarknessSourceAnimationConfig>;
 
         pings: {
             types: {
@@ -694,6 +706,9 @@
     /** Available Weather Effects implemntations */
     weatherEffects: Record<string, WeatherAmbienceConfiguration>;
 
+    /** Sound effects which can be applied to Sound instances */
+    soundEffects: Record<string, { label: string; effectClass: ConstructorOf<AudioNode> }>;
+
     /** Configuration for dice rolling behaviors in the Foundry VTT client */
     Dice: {
         types: (typeof dice.terms.Die | typeof dice.terms.DiceTerm)[];
