diff --git a/client/documents/_types.d.mts b/client/documents/_types.d.mts
index 506f080..b0ddb97 100644
--- a/client/documents/_types.d.mts
+++ b/client/documents/_types.d.mts
@@ -77,7 +77,6 @@ export type AdventurePostImportCallback = (
 export interface ActiveEffectChangeData<
     TEffect extends ActiveEffect<any> = ActiveEffect<any>,
 > extends EffectChangeData {
-    key?: string;
     effect?: TEffect;
     priority: number;
 }
diff --git a/common/documents/active-effect.d.mts b/common/documents/active-effect.d.mts
index 533120f..d295584 100644
--- a/common/documents/active-effect.d.mts
+++ b/common/documents/active-effect.d.mts
@@ -82,6 +82,8 @@ type ActiveEffectSchema = {
 };
 
 type EffectChangeSchema = {
+    /** The attribute path in the Actor or Item data which the change modifies */
+    key: fields.StringField<string, string, true, false, true>;
     type: fields.StringField<string, string, true, false, true>;
     value: fields.AnyField;
     phase: fields.StringField<string, string, true, false, true>;
